docs/excel/custom-functions-authentication.md

```diff
@@ -1,52 +1,46 @@
 ---
-ms.date: 07/09/2019
-description: 'Authenticate users using custom functions in Excel.'
-title: Authentication for custom functions
+ms.date: 05/13/2020
+description: 'Authenticate users using UI-less custom functions in Excel.'
+title: Authentication for UI-less custom functions
 localization_priority: Normal
 ---
 
-# Authentication for custom functions
+# Authentication for UI-less custom functions
 
```

Comment:
I feel like we should have some note above all the UI-less custom functions articles pointing people to the shared runtime. Or do we think people should still opt to use this pattern with equal frequency?